### DISCUSSION TITLE
How to document between to applcations

### DISCUSSION THREAD

**Discussion by lajaunie** [Alef-Burzmali/netbox-data-flows#40] [General]
Hello,
i read the Quick Start tutorial and i don't understand how to document a data flow between two applications.
In my mind, this particular Data Flow is the same in both applications.
I try to use data-flow-groups, but a data-flow rule belong to only one group.
Did i need to duplicate this data flow-rule in both applications ?

thank you

**Comment by Alef-Burzmali** [comment]
Hello,
Yes, it is a limitation of the plugin, you cannot associate a data flow to two different applications. Here are some suggestions that may work for you:
- you can chose to map the data flow to the predominant application (I usually go for the server side), especially if it's an API that is exposed or similar
- if the applications have many data flows in common, you can create a single application that regroups them both, or (better in my opinion), a data flow group without an application that contains all the data flows in common between these two apps
- for applications that represent a shared service used by many other applications (such as a web proxy), I usually create a data flow group that contains all the data flows to the shared service, and each data flow is mapped to the application that requires it
- you can also map the data flow to the application that really requires the flow and would not work without it (hopefully it's not both applications)

If there is a significant demand for data flows mapped to several applications, I may review the model relationship, but it's a non trivial change.
